IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs

```diff
@@ -849,6 +843,68 @@ private void NoButton_Click(object sender, RoutedEventArgs e)
         }
 
         /// <summary>
+        /// Turn on Bluetooth Radio and list available Bluetooth Devices
+        /// </summary>
+        private async void TurnOnRadio()
+        {
+            await ToggleBluetoothAsync(true);
+            SetupBluetooth();
+        }
+
+        /// <summary>
+        /// Checks the state of Bluetooth Radio 
+        /// </summary>
+        private async Task<bool> IsBluetoothEnabledAsync()
+        {
+            var radios = await Radio.GetRadiosAsync();
+            var bluetoothRadio = radios.FirstOrDefault(radio => radio.Kind == RadioKind.Bluetooth);
+            return bluetoothRadio != null && bluetoothRadio.State == RadioState.On;
```

Comment:
The bluetoothRadio.State saves the systems state if the radio is turned on. The function checks for this and accordingly displays the Bluetooth settings.